templates/cli/lib/commands/deploy.js.twig

```diff
@@ -264,6 +224,47 @@ const deployFunction = async ({ functionId, all, yes } = {}) => {
             }
         }
 
+        // Create function variables
```

Comment:
This is logic for initial creation. We can remove a lof of code - we don't need to list, ask user if he is sure, and delete..

We could simply run this here:

```php
if(func.variables) {
            // Deploy local variables
            await Promise.all(Object.keys(func.variables).map(async localVariableKey => {
                await functionsCreateVariable({
                    functionId: func['$id'],
                    key: localVariableKey,
                    value: func.variables[localVariableKey],
                    parseOutput: false
                });
            }));
}
```